integration/src/main/resources/suites/FullIntegration.json

```diff
@@ -16,6 +16,7 @@
     "integration/EnumerateDataReferences.json",
     "integration/EnumerateResources.json",
     "integration/GetRoles.json",
+    "integration/ListWorkspaces.json",
```

Comment:
Could you also add this line to [dev](https://github.com/DataBiosphere/terra-workspace-manager/blob/main/integration/src/main/resources/suites/dev/FullIntegration.json), [alpha](https://github.com/DataBiosphere/terra-workspace-manager/tree/main/integration/src/main/resources/suites/alpha), and [staging](https://github.com/DataBiosphere/terra-workspace-manager/tree/main/integration/src/main/resources/suites/staging) suites? We should be able to avoid duplicates like this when [QA-1460](https://broadworkbench.atlassian.net/browse/QA-1460) is done.

Reply:
done